test/vttest_sample_test.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+# Copyright 2015 Google Inc. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can
+# be found in the LICENSE file.
+
+"""This sample test demonstrates how to setup and teardown a test
+database with the associated Vitess processes. It is meant to be used
+as a template for unit tests.
```

Comment:
You might want to clarify that it's for unit testing the app you're developing against Vitess, not just for Vitess's own tests.